reascripts/ReaSpeech/source/ASRPlugin.lua

```diff
@@ -0,0 +1,87 @@
+--[[
+
+  ASRPlugin.lua - ASR plugin for ReaSpeech
+
+]]--
+
+ASRPlugin = Polo {
+  new = function(app)
+    return {
+      app = app
+    }
+  end,
+}
```

Comment:
similar to how ancillary plugin classes refer back to their owning plugin, a plugin gets a reference to the host app (here, the instance of `ReaSpeechUI` that we've been usually accessing via the global `app`)